### DISCUSSION TITLE
Streamdeck Studio API

### DISCUSSION THREAD

**Discussion by maccalder** [elgatosf/elgato#151] [Help]
Now that Elgato has the Studio - which can be interfaced via ethernet - is the ip api available publicly? rather than using the SDK over USB?

**Comment by ChekTek** [comment]
Our team is currently evaluating the release of the IP API. How are you wanting to use Stream Deck Studio?

**Reply by maccalder** [reply]
I'd like to interface with a Crestron or Qsys control systems (as well as our inhouse, proprietary control system) without the need for middleware.

**Reply by ChekTek** [reply]
Thanks! I will take this feedback to the Stream Deck team.

**Comment by JaredVogt** [comment]
I also would like to use IP API. I want to use it as a controller for analog audio patching matrix with a http based api. Everything is running over ethernet and POE - so this would be perfect.

**Comment by ChekTek** [comment]
Closing this issue, as it has been reported as a feature request to the Stream Deck team.
